contracts/UnitPricesGovernance.sol

```diff
@@ -49,10 +50,10 @@ contract UnitPricesGovernance is IUnitPricesGovernance, DefaultAccessControl {
 
     /// @inheritdoc IUnitPricesGovernance
     function commitUnitPrice(address token) external {
+        _requireAdmin();
```

Comment:
@alexeuler 
noticed missing `_requireAdmin()` here 
though maybe it's OK not to check admin permission here and let anyone commit when delay passes

Reply:
I think that can make sense to avoid unnecessary governance management. Let's keep it for now and then discuss to probably remove commit access control everywhere

Reply:
That can make sense to avoid unnecessary governance management. Let's keep it for now and discuss to remove this access control for anything else.